### PR TITLE
Adds Browser Support Policy Page

### PR DESCRIPTION
**Pull request by pzuraq**
Adds a browser support policy page, as described in the [new browser support policy RFC](https://github.com/emberjs/rfcs/blob/master/text/0685-new-browser-support-policy.md).

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -56,6 +56,8 @@ Router.map(function () {
     this.route('release');
   });
 
+  this.route('browser-support');
```
Comment (ijlee2) [inline app/router.js:59]: Could you place this line between the lines for `blog` and `community`? I'd like to keep the list alphabetized to help with maintenance. 🙂

File: app/templates/browser-support.hbs [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  <section aria-labelledby="section-browser-support-policy">
+    <h1 id="section-browser-support-policy">Ember.js Browser Support Policy</h1>
+
+    <p>
+      Ember supports the following major browsers:
+    </p>
+
+    <ul class="list-unstyled layout my-3">
+      <EsCard class="lg:col-2">
+        <h3 class="text-center">Desktop</h3>
```
Comment (ijlee2) [inline app/templates/browser-support.hbs:12]: I think using `<h3>` right after `<h1>` will fail accessibility audit. Can we use a different tag? (We can always customize styling.)

**Review by ijlee2 — CHANGES_REQUESTED** [review]
Hi, Chris. I checked the deploy preview website and thought the content and layout looked great.

Can you update the PR by:

- Adding 2 application tests (Percy snapshot and accessibility audit) for the `browser-support` route? You can copy-paste the `tests/acceptance/community-test.js` file into `tests/acceptance/browser-support-test.js` file, for example.

I believe the `browser-support` route won't be discoverable unless we provided people a direct link.

If we want the page to show up on the top navigation menu, we'll need to make a pull request in `ember-styleguide` repo and update the addon in this project. (There was a change to the nav menu that I wanted to make, so it may be a good idea to combine both tasks into one PR.)

Screenshot 2021-01-26 at 12 45 00

A potential problem may be that we don't currently use the latest version of `ember-styleguide` (`v5`). We'd need to coordinate with the maintainers of `ember-styleguide` to see if they can make a minor release in `v4`.

**Comment by pzuraq** [comment]
Awesome, yep, sounds good 😄  I wasn't sure what the procedure was for adding top level links so I figured we could figure it out. Will update with the changes, and should I make a PR to `ember-styleguide`?

**Review by ijlee2 — APPROVED** [review]
Thank you again for helping out with describing the new Browser Support policy!

**Pull request merged**